pkg/sql/parser/sql.y

```diff
@@ -6579,12 +6617,36 @@ alter_rename_view_stmt:
     newName := $6.unresolvedObjectName()
     $$.val = &tree.RenameTable{Name: name, NewName: newName, IfExists: false, IsView: true}
   }
+| ALTER MATERIALIZED VIEW relation_expr RENAME TO view_name
+  {
+    name := $4.unresolvedObjectName()
+    newName := $7.unresolvedObjectName()
+    $$.val = &tree.RenameTable{
+      Name: name,
+      NewName: newName,
+      IfExists: false,
+      IsView: true,
+      IsMaterialized: true,
+    }
+  }
 | ALTER VIEW IF EXISTS relation_expr RENAME TO view_name
```

Comment:
can we just do
```
| ALTER opt_materialized VIEW IF EXISTS ...
    name := $6.unresolvedObjectName()
    newName := $9.unresolvedObjectName()
    $$.val = &tree.RenameTable{Name: name, NewName: newName, IfExists: true, IsView: true, IsMaterialized: $2.bool()}
```
?

Reply:
lol i tried this, but got some nice shift/reduce conflicts so I gave up.

Reply:
hmmMmMmMMMMmm